applications/crossbar/priv/couchdb/views/rates.json

```diff
@@ -3,10 +3,22 @@
     "language": "javascript",
     "views": {
         "crossbar_listing": {
-            "map": "function(doc) { if (doc.deleted || doc.pvt_type !== 'rate') return; emit(doc._id, {\"name\":doc.rate_name, \"cost\":doc.rate_cost, \"prefix\":doc.prefix, \"description\":doc.description, \"surcharge\":doc.rate_surcharge, \"carrier\":doc.carrier, \"id\":doc._id}); }"
+            "map": [
+                "function(doc) {",
+                "  if ( doc.pvt_deleted || doc.pvt_type !== 'rate') return;",
+                "  emit(doc._id, {",
+                "\"name\":doc.rate_name,",
```

Comment:
Keys don't need to be quoted. If you really want to quote them you can use `'`.
Formatting: please indent & add a space after `:`